0002_child_test.js

```diff
@@ -321,23 +347,23 @@ describe("Test Social Worker Note:", () => {
       cy.log(
```

Comment:
According to this comment:
If the popup is closing, these should be deleted as part of the change that fixes the proble